modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/edit_kb_article.jsp

```diff
@@ -359,21 +359,26 @@ if (editKBArticleDisplayContext.isPortletTitleBasedNavigation()) {
 	</div>
 </aui:form>
 
-<portlet:renderURL var="scheduleModalURL" windowState="<%= LiferayWindowState.POP_UP.toString() %>">
-	<portlet:param name="mvcPath" value="/admin/common/schedule_modal.jsp" />
-	<portlet:param name="displayDate" value="<%= editKBArticleDisplayContext.getDatePickerFormattedDisplayDate() %>" />
-	<portlet:param name="scheduled" value="<%= String.valueOf(editKBArticleDisplayContext.isScheduled()) %>" />
-</portlet:renderURL>
-
 <liferay-frontend:component
 	context='<%=
 		HashMapBuilder.<String, Object>put(
 			"kbArticle", editKBArticleDisplayContext.getKBArticle()
 		).put(
 			"publishAction", WorkflowConstants.ACTION_PUBLISH
-		).put(
-			"scheduleModalURL", scheduleModalURL.toString()
 		).build()
 	%>'
 	module="admin/js/EditKBArticle"
-/>
\ No newline at end of file
+/>
+
+<div>
+	<react:component
+		module="admin/js/components/ScheduleKBArticle"
+		props='<%=
+			HashMapBuilder.<String, Object>put(
+				"displayDate", editKBArticleDisplayContext.getDatePickerFormattedDisplayDate()
+			).put(
+				"isScheduled", editKBArticleDisplayContext.isScheduled()
```

Comment:
Boolean variable names should not have the `is*` prefix. It should be simply "scheduled". The `is*` prefix is only used for methods.

Reply:
Renamed! Thanks @adolfopa